tests/boards/intel_adsp/hda/src/tests.h

```diff
@@ -15,6 +15,23 @@ void test_hda_host_in_smoke(void);
 void test_hda_host_out_smoke(void);
 void test_hda_host_in_dma(void);
 
+
+/* Turn this define on to see register dumps after each step */
+#define INTEL_ADSP_HDA_DBG 0
```

Comment:
nitpick. This could be a Kconfig option or an overlay to the sample, then we wouldn't need to edit the code to get it.